src/test/utils/objectKeyPaths.test.js

```diff
@@ -30,6 +34,25 @@ describe('getKeyPathsArray()', () => {
       ],
     ]);
   });
+
+  test('prop w/ version should have version removed when stripVersions flag is set', () => {
```

Comment:
Not a blocker, but does this test that '_v' followed by non-digits or periods don't get stripped?